widgetssdk/src/main/java/com/glia/widgets/call/CallState.java

```diff
@@ -22,6 +22,8 @@ class CallState {
     public final Engagement.MediaType requestedMediaType;
     public final boolean isSpeakerOn;
     public final boolean isOnHold;
+    //    Need this to not update all views when only time is changed.
+    public final boolean isOnlyTimeChanged;
```

Comment:
It might be just me but I didn't understand this nither from the comment or the code implementation.

Reply:
I guess it is for the count-up timer.

Reply:
Every screen in our `Widgets SDK` has its own `state`(for `call screen` it is `CallState`), and based on that state screen is rendering widgets(`CallView.emitState(CallState)`). On `Call Screen`, we have a call timer and to update time on it, we are updating CallState every second, which is causing an entire screen update instead of updating only the timer. So I added this field which is indicating that only time has changed, and no need to update the remaining widgets' state.
If you have a suggestion on how to write a more clear comment I will change it